src/Traits/ReflectionFunctionLikeTrait.php

```diff
@@ -53,7 +60,7 @@ public function getClosureScopeClass()
     {
         $this->initializeInternalReflection();
 
-        return forward_static_call('parent::getClosureScopeClass');
+        return parent::getClosureScopeClass();
```

Comment:
Why change this?

Reply:
This is just blatantly wrong. This is an instance method, not a static method.

Reply:
Why this was working before and needs to be changed now? Maybe this was workaround for older PHP versions?

Reply:
👍  for moving this into separate PR.

Reply:
One other thing, for **this** PR: the native implementation returns `?ReflectionClass`, so the result needs to be reinstantiated as a `Go\ParserReflection\ReflectionClass` before it's returned, unless it's `null`.

Reply:
No big deal. Good catch.

Reply:
Ok... doesn't appear that the old code has any bad behavior, but I moved this into PR #86.